app/controllers/v1/bookings_controller.rb

```diff
@@ -26,6 +28,15 @@ def create
       end
     end
 
+    def destroy
+      if @booking.present?
+        @booking.destroy
+        render json: { message: 'Booking Cancelled Successfully!' }, status: 200
```

Comment:
I think we can add status column in booking table with default value as 'booked' and change the status to 'cancel' when the booking is cancelled.
@kirti09

Reply:
Not sure whether we need to cancel the booking - we do not need to actually store the state of the booking here. If it is cancelled then the meeting room is available to be booked at the slot. @syedtabrez123 is there is any compelling reason that we need to store that a booking has been cancelled?